### PR TITLE
F11 textutility

### PR DESCRIPTION
**Pull request by sushmita12345**
Text Utilities, Navigation, Snackbar, Grid is added to Component library.

Sample
Screenshot 2022-02-08 at 12 26 01 AM
Screenshot 2022-02-08 at 12 26 20 AM
Screenshot 2022-02-08 at 12 26 54 AM

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]

File: components/docs.html [hunk 1]
```diff
@@ -0,0 +1,1104 @@
...
+                                </div>
+                                <div class="btn-left">
+                                    <button class="nav-btn">Login</button>
+                                    <span class="material-icons nav-heart-icon">
+                                        favorite_border
+                                    </span>
+                                    <span class="nav-cart-icon">
+                                        <span class="material-icons">
+                                            shopping_cart
+                                        </span>Cart
```
Comment (nisargpawade) [inline components/docs.html:968]: I think you can remove the text `Cart`

**Review by nisargpawade — APPROVED** [review]
everything else looks fine

**Comment by deekshithmd** [comment]
code looks good

File: components/docs.html (hunk 1 above)
Comment (sushmita12345) [reply components/docs.html:968]: Thanks, changes has been done.

**Pull request merged**